File: Driver-App/app/src/main/java/com/tkpmnc/newtaxidriver/home/interfaces/ServiceListener.java
package com.tkpmnc.newtaxidriver.home.interfaces;

import com.tkpmnc.newtaxidriver.common.model.JsonResponse;


public interface ServiceListener {

    void onSuccess(JsonResponse jsonResp, String data);

    void onFailure(JsonResponse jsonResp, String data);


    /*void onSuccessResponse(JsonResponse jsonResp, String data);*/
}

File: User-App/app/src/main/java/com/tkpmnc/newtaxiusers/appcommon/interfaces/ServiceListener.kt
package com.seentechs.newtaxiusers.appcommon.interfaces

/**
 *  newtaxiusers
 * @subpackage interfaces
 * @category ServiceListener
 *  
 * 
 */

import com.tkpmnc.newtaxiusers.appcommon.datamodels.JsonResponse

/*****************************************************************
 * ServiceListener
 */
interface ServiceListener {

    fun onSuccess(jsonResp: JsonResponse, data: String)

    fun onFailure(jsonResp: JsonResponse, data: String)
}
